src/constants/config.json

```diff
@@ -4407,6 +4407,19 @@
 				}
 			]
 		},
+		{
```

Comment:
@praveenKDass for api with :id we might need config without id also. I have noticed it please check once

Reply:
Added without id @VISHNUDAS-tunerlabs